common/api_client.go

```diff
@@ -1,15 +1,14 @@
-package nozzle
+package common
```

Comment:
It's encouraged in Go to name packages after the function they provide instead of a `common` pkg with everything in it. See the section on "bad package names" https://blog.golang.org/package-names.

Might be better to have an `internal` package with multiple smaller packages within it say `application`, `config`, `filter` etc